public/app/plugins/panel/timeseries/models.cue

```diff
@@ -0,0 +1,42 @@
+// Copyright 2021 Grafana Labs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package grafanaschema
+
+import (
+    ui "github.com/grafana/grafana/cue/ui:grafanaschema"
+)
+
+Family: {
+    lineages: [
+        [
+            {
+                PanelOptions: {
+                    legend: ui.VizLegendOptions
+                    tooltip: ui.VizTooltipOptions
+                }
+                PanelFieldConfig: {
+                    ui.LineConfig
```

Comment:
Are you planning to create object in gen.cue and reference those objects in each models.cue panels?